test/f90_correct/src/qc_abs_01.f08

```diff
@@ -0,0 +1,79 @@
+!** Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+!** See https://llvm.org/LICENSE.txt for license information.
+!** SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+!*  Test intrinsics function abs with quad precision complex.
+
+program main
+  use check_mod
+  implicit none
+  integer, parameter :: k = 16
+  integer, parameter :: n = 15
+  complex(kind=k), parameter :: t1 = (1.23434590874573456543454E+30_16, &
```

Comment:
Are the test input data randomly selected? Did you include any boundary values such as maximum and minimum quad-precision values? Please add a comment to elaborate on the data selection.

Reply:
This applies to the test data in the other test cases as well.

Reply:
I have refactored these test cases and PR has been updated, please have a look

Reply:
@wanbinchen-hnc Did you push your updates correctly? When I try to review the differences, GitHub reports that "ae192dc9b254a9eda5d97266a17c3446ee7f667d and fde217fc7b80f0d68f6dbc1ca02c10dba70993f6 are identical".

Reply:
Thanks for your reminding, I have pushed my updates again.